src/modules/site-v2/base/views/tools/phenotype_database.py

```diff
@@ -171,17 +171,17 @@ def get_traits_json():
     Get traits data for non-bulk files in JSON format (include phenotype values)
   """
   trait_name = request.json.get('trait_name')
+  err_msg = f'Failed to retrieve metadata for trait {trait_name}'
 
   if trait_name:
     try:
       trait = get_trait(trait_name).to_json_with_values()
       return jsonify(trait)
 
     except Exception as ex:
-      msg = f'Failed to retrieve metadata for trait {trait_name}'
-      logger.error({msg: ex})
+      logger.error({err_msg: ex})
```

Comment:
I thought of mentioning this here since the behavior is different than `js`. The keys for python dictionaries are an `expression` type that 'evaluate' to the `key` to be used -- unless they are under quotes, or are a symbol/etc. For instance:
```python
 >>> a = 'key'
>>> b = { a: 42}
>>> b
{'key': 42}
```

Reply:
with that in mind, consider having a key `error` in the resulting object, and concatenate the exception message to it.